models/articles.models.js

```diff
@@ -12,4 +12,18 @@ const selectArticleById = (article_id) => {
     });
 };
 
-module.exports = { selectArticleById };
+const selectAllArticles = () => {
+  return db
+    .query(
+      `SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.article_id) AS comment_count
+  FROM articles
+  LEFT JOIN comments ON articles.article_id = comments.article_id
+  GROUP BY articles.article_id
+  ORDER BY articles.created_at DESC;`
+    )
+    .then(({ rows }) => {
+      return rows;
+    });
+};
```

Comment:
Great use of the query string to get your comment count and make it a number instead of a string 🤸